File: Unidad 2-Estructuras de Control/Ejercicio14.kt
//14.Escribir el programa que lea una velocidad e imprima “Muy rápido” si es superior a 100,
// si no imprima “Detenido” si la velocidad es cero.
//Autor: Rodrigo Hernández Góngora
fun main(args: Array<String>){
    //Entradas: Un número entero (velocidad)
    println("Introduce la velocidad")
    var vel= readLine()!!.toInt()
    //Proceso
    if (vel>=100){
        println("Muy rapido")
    }
    else if (vel==0){
        println("Detenido")
        //Salidas: La oracion correspondiente con el dato introducido
    }
}
/*
QA: Hector Cosgalla
Entradas: Un numero entero (velocidad)
Salidas: la evaluación de los numeros en sus respectivos casos
Comentarios: No lee el rango de velocidades de 1 a 99
*/

File: Unidad 2-Estructuras de Control/Ejercicio14.c
//Autores: Programa realizado por el Equipo2_CodePain
//Version 1.5
//Este programa lee una velocidad y determina si es mayor a 100, imprime "muy rapido" y si es cero, imprime "detenido"

#include <stdio.h>

int main(){

//Entrada: Un número flotante vel
float vel=0;

scanf("%f", &vel);

//Proceso: Checar si la velocidad es mayor a 100 o igual a 0
//Salida: "Muy rapido" o "Detenido" o nada si no es cero o mayor a cien

if (vel<0) {
    printf("Ingrese una velocidad válida");
} else {
    if (vel>100) {
        printf("Muy rápido");
    } else {
        if (vel==0) {
            printf ("Detenido");
        }
    }
}
}